src/client.ts

```diff
@@ -1795,43 +1791,43 @@ export class StreamChat<
   /**
    * updateMessage - Update the given message
    *
-   * @param {MessageResponse<MessageType, AttachmentType, ChannelType, ReactionType, UserType, CommandType>} message object, id needs to be specified
```

Comment:
don't need the rest of these Types anymore?

Reply:
previous paste error in the comment they don't type check so we missed it before.